File: SEMESTER 2/Java/Cycle 1/SimpleInterest.java
import java.util.*;
class SimpleInterest
{
	public static void main(String args[])
	{
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter principle amount: ");
		float p = sc.nextFloat();
		System.out.println("Enter number of years: ");
		float n = sc.nextFloat();
		System.out.println("Enter rate of interest: ");
		float r = sc.nextFloat();

		float si = (p * n * r) / 100;
		System.out.println("Simple Interest: " + si);
	}
}

File: SEMESTER 1/C/Cycle 1/simpleInterest.c
#include <stdio.h>
int main()
{
	int n;
	float p, r, si;
	printf("Enter principle amount: ");
	scanf("%f", &p);
	
	printf("Enter number of years: ");
	scanf("%d", &n);
	
	printf("Enter rate of interest: ");
	scanf("%f", &r);
	
	si = (p * n * r)/100;
	
	printf("Simple interest: %.2f\n", si);	
}
